Contribution section (#16)
rs-ipfs/offchain-ipfs-manual: src/contributing-development.md

# Contributing / Development

There are several ways that people can help `offchain::ipfs`.

> ## Create a dApp or custom Substrate pallet, and tell us about it

Reviewer: ```suggestion
## Create a dApp or a custom Substrate pallet, and tell us about it
```

Now that the proof-of-concept is complete and this manual is published, our goal is to be

> laser-focused on user needs and to build out the capabilities of `offchain::ipfs` based on those.

Reviewer: ```suggestion
laser-focused on user needs and to extend the capabilities of `offchain::ipfs` based on them.
```

> So, **if you end up trying this out, we would love to know how, and for what.**

Reviewer: ```suggestion
So, **if you end up trying this out, we would love to know how, and what your use case is.**
```

- Tell us about what you're doing by opening an information issue at [rs-ipfs/substrate]
- Share feedback on this manual by opening an issue at [rs-ipfs/offchain-ipfs-manual]

> - Share feedback on `offchain::ipfs` itself by open an issue at [rs-ipfs/substrate]

Reviewer: ```suggestion
- provide feedback on `offchain::ipfs` itself by opening an issue at [rs-ipfs/substrate]
```

## Support `offchain::ipfs` financially

> [Equilibrium] is a consultancy that builds, and helps you build _on_, the distributed web. If

Reviewer: ```suggestion
[Equilibrium] is a consultancy that builds, and helps you build _on_ the distributed web. If
```
I know what the purpose of that extra comma was, but the sentence is pretty short, so it felt odd

you're part of an organization and want help implementing or improving `offchain::ipfs`,
get in touch with us.

- If you're an individual and want to support us financially, visit our [OpenCollective].

## Implement more IPFS functionality

> Some of the functionality exists in [Rust IPFS] and is not exposed via `offchain::ipfs`, and some

Reviewer: ```suggestion
Some of the functionality that exists in [Rust IPFS] is not exposed via `offchain::ipfs`, and some
```

> of the functionality is not implemented in Rust IPFS at all. If there's something that another

Reviewer: ```suggestion
of the IPFS functionalities are not implemented in Rust IPFS at all. If there's something that another
```
